jest.config.js

```diff
@@ -0,0 +1,184 @@
+// For a detailed explanation regarding each configuration property, visit:
```

Comment:
most of the commented out stuff was generated via the CLI. we don't need to keep it in, but I left it in for context and some light reading material